internal/osv2/mget.go

```diff
@@ -0,0 +1,152 @@
+package osv2
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/opensearch-project/opensearch-go/v2/opensearchapi"
+
+	"github.com/CrowdStrike/opensearchtools"
+)
+
+// The main purpose of this file is to implement the MGet interface on the OSv2Executor.
+// In order to do this, it also defines a bunch of marshalable types for making requests and getting
+// responses from OpenSearch 2. These might have a different shape than the similarily named ones in the root mget.
+
+// MGet executes the Multi-Get MGetRequest using the provided opensearchtools.MGetRequest.
```

Comment:
General note on doc comments: use the new link syntax - `[opensearchtools.MGetRequest]` here - when referencing types so that the docs on pkg.go.dev will be cross linked.

More details on the syntax [here](https://tip.golang.org/doc/comment).

Reply:
Thanks - fixed this up and elsewhere in the files this PR touches.